sandbox/functions-recipes/includes/queue-storage-queue-trigger-with-poco.md

```diff
@@ -0,0 +1,22 @@
+## Azure Storage Queue Trigger using a POCO
```

Comment:
Should we define POCO here (I had no idea what it was) @anthonychu @cecilphillip?

Reply:
Yes. I agree we should have a POCO here.